vulture/core.py

```diff
@@ -415,6 +420,37 @@ def visit(self, node):
             visitor(node)
         return self.generic_visit(node)
 
+    def _handle_ast_list(self, ast_list):
+        """
+        Iterates over the given list of ast nodes and finds unreachable code,
+        like any code after return statemnts.
+        """
+        for index, node in enumerate(ast_list):
+            if isinstance(node, ast.Return):
+                try:
+                    first_unreachable_node = ast_list[index + 1]
+                except IndexError:
+                    continue
+                self._define(
+                    self.unreachable_code,
+                    'return',
+                    first_unreachable_node.lineno,
+                    size=(
+                        lines._get_last_line_number(ast_list[-1]) -
```

Comment:
I misunderstood and thought that this change has been done upstream and I need to pull it in. :p 

Done!